frontend/src/Components/Schedule/Schedule.jsx

```diff
@@ -13,24 +13,53 @@ const ScheduleRow = React.memo(({ item, isCurrentEvent }) => {
 
   const currentTime = new Date().getTime();
   const isPastEvent = currentTime > item.endTime.getTime();
+  const hour = 3600000;
+  const isCurrentEventv2 = item.startTime.getTime() <= currentTime && currentTime <= item.endTime.getTime()
+  const isUpcomingEvent = item.startTime.getTime() <= (currentTime + hour) && !isPastEvent
 
   return (
     <tr
       style={{
+        height: '70px',
```

Comment:
this literally does nothing, was tinkering and forgot to delete (figured it wasn't worth making a whole commit to delete 10 characters)